src/editor/tools/local-autosave.js

```diff
@@ -1,6 +1,5 @@
 import db from "./local-autosave-client";
```

Comment:
I am thinking perhaps we should just inline the code in the redux actions.

Reply:
I think I'm just going to leave it for now, on second thought. I think it's better to abstract away some of the details of accessing indexedb from the redux actions.